src/agent/open_monitor.rs

```diff
@@ -324,7 +351,7 @@ async fn monitor_fanotify(fan: Arc<Fanotify>, probes: BpfProbes, ch: Sender<Open
                 }
             };
 
-            let cgroup_name = match probes.lookup_cgroup(e.pid as u32) {
+            let cgroup_name = match probes.lock().unwrap().lookup_cgroup(e.pid as u32) {
```

Comment:
Is this unwrap safe? I see this pattern in a couple places.

Reply:
Yes, it's safe. `lock()` will fail if another thread paniced while holding the lock. In this case I view it as a fatal event anyway so this thread is welcome to panic.